NewForce-Cohort-9/tabloid-full-stack-full-stack-force
# Post Details

# What changed?

Added the ability to click on the name of a post to view its details.

# How to test?

1. Add and commit any changes on your own branch
2. git checkout main
3. git fetch --all
4. git checkout tylerBranch-postDetails
5. git pull origin tylerBranch-postDetails
6. You've certainly made your SQL database by now, right?
7. Run your API 
8. Activate the app using your terminal of choice
9. Log in
10. Navigate to either "Posts" or "My Posts"
11. Click a post, view the details.  Pictures don't seem to be working but that's okay.  You'll see the actual contents of the post.
12. Go and click another post (Why not try one from the opposite page as you chose on step 10? What fun.)
13. Click the name and view the details.
14. Congratulations, you have viewed details.
